make/run-cicd-script.mk

```diff
@@ -1,4 +1,4 @@
-OWNER_AND_BRANCH_LOCATION=codeready-toolchain/toolchain-cicd/master
+OWNER_AND_BRANCH_LOCATION=mfrancisc/toolchain-cicd/feature/ASC-256_sr_basic
```

Comment:
reminder

Reply:
Fixed in [cadc8f4](https://github.com/codeready-toolchain/host-operator/pull/751/commits/cadc8f46cf295554139add65acd621791cda8674)